p256k1/src/field.rs

```diff
@@ -25,13 +25,27 @@ pub const P: [u8; 32] = [
     0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFE, 0xFF, 0xFF, 0xFC, 0x2F,
 ];
 
+/// Re-export of crate `bs58`'s decode error
```

Comment:
Since you've added this to both `point` and `scalar`, it's probably worth making an `errors.rs` file and putting this and the existing `ConversionError` there.  If you need to you can typedef it in `scalar` and `point`.